### PR TITLE
Add downloaded icon in TransitionView when chapter is downloaded

### PR DESCRIPTION
**Pull request by nzoba**
Add icon in the `TransitionView` to know if the next/previous chapter is downloaded without exiting the reader (almost like in the J2K [PR](https://github.com/Jays2Kings/tachiyomiJ2K/pull/1174)). The icon is visible next to the chapter's name if it is downloaded.

TransitionView icon

### CODE REVIEW

**Comment by Riztard** [comment]
what happen if the chapter title is very long

**Comment by nzoba** [comment]
It adds a new line, the icon is just appended to the end of the chapter title like text.

TransitionView icon with long title

**Comment by CrepeTF** [comment]
I recommend using a different icon since this one implies an action. The icon named offline_pin would work better imo

**Comment by nzoba** [comment]
Sure, I don't mind

TransitionView changed icon

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/ReaderTransitionView.kt [hunk 1]
```diff
@@ -21,31 +30,44 @@ class ReaderTransitionView @JvmOverloads constructor(context: Context, attrs: At
         layoutParams = LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT)
     }
 
-    fun bind(transition: ChapterTransition) {
+    fun bind(transition: ChapterTransition, downloadManager: DownloadManager, manga: Manga?) {
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/ReaderTransitionView.kt:33]: This definitely feels a bit weird in how it's being passed around, but it'll probably end up getting rewritten at some point soon anyhow.

**Pull request merged**